Dockerfile

```diff
@@ -13,14 +13,18 @@ ENV TITLE=FreeCAD
 RUN \
   echo "**** add icon ****" && \
   curl -o \
-    /kclient/public/icon.png \
-    https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/freecad-logo.png && \
-  echo "**** install packages ****" && \
-  apt-get update && \
-  apt-get install -y --no-install-recommends \
-    freecad \
-    python3-pyside2.qtwebchannel \
-    python3-pyside2.qtwebengine* && \
+  /usr/share/icons/hicolor/48x48/apps/freecad.png \
+  https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/freecad-logo.png
+RUN \
+  echo " install freecad from appimage " && \
+  freecad_version="1.0.0" && \
+  cd /tmp && \
+  curl -o /tmp/freecad.app -L https://github.com/FreeCAD/FreeCAD/releases/download/1.0.0/FreeCAD_1.0.0-conda-Linux-x86_64-py311.AppImage && \
```

Comment:
We are not going to accept a hard coded version number.

Reply:
replaced with environment variable